src/__private_api.rs

```diff
@@ -34,6 +34,14 @@ impl<'a> KVs<'a> for () {
     }
 }
 
+pub fn file<'a>() -> &'a str {
+    ::std::panic::Location::caller().file()
+}
+
+pub fn line() -> u32 {
+    ::std::panic::Location::caller().line()
+}

```

Comment:
I think this requires `#[track_caller]` in order to work correctly.

Reply:
🤦 yep, apparently it's been too long since I did this. Thanks for the catch

Reply:
@EFanZh this is done